pkg/drivers/kvm/network.go

```diff
@@ -124,8 +124,8 @@ func (d *Driver) lookupIPFromStatusFile(conn *libvirt.Connect) (string, error) {
 		return "", errors.Wrap(err, "reading status file")
 	}
 	type StatusEntry struct {
-		IPAddress string `json:"ip-address"`
-		Hostname  string `json:"hostname"`
```

Comment:
This doesn't look like the correct approach.

Each VM should set their hostname according to the machinename provided. In the default case this would be 'minikube', but in your example 'machine1' and 'machine2'

Reply:
https://github.com/kubernetes/minikube/blob/f5d945597d46b17bd6a6683206801e540113de9b/pkg/provision/buildroot.go#L145-L148

So the value passed is wrong... ?